support/origins-starter-app/src/main/java/com/hotels/styx/support/origins/AppHandler.java

```diff
@@ -57,11 +61,11 @@ public AppHandler(Origin origin) {
                     request.queryParam("status").ifPresent(status ->
                             responseBuilder
                                     .status(httpResponseStatus(status))
-                                    .body(StyxObservable.of("Returning requested status (" + status + ")"), UTF_8)
+                                    .body(new ByteStream(toPublisher(Observable.just(new Buffer("Returning requested status (" + status + ")", UTF_8)))))
```

Comment:
As in other parts of coude, Is there any reason (visibility?) why we cannot use Flux.just() instead of creating an rxJava Observable with .just()  and transforming it to a publisher  ?

Reply:
Yeah it looks it is OK to use Flux.

Reply:
I didn't want to add the comment in every occurrence but you can search for Observable.just()...